### ISSUE TITLE
Scrollstories d3.js TypeError: null is not an object

### ISSUE THREAD

**Issue by lukasniebler** [RRZE-Webteam/FAU-Einrichtungen#1357]
![Bildschirmfoto 2023-06-22 um 14 00 47](https://github.com/RRZE-Webteam/FAU-Einrichtungen/assets/58593037/3c6f62f4-e899-43f3-a83b-1f11dae7d3c0)

Wenn man eine Inhaltsseite mit Template Scrollstories anlegt, wirft die Konsole den obigen Fehler, wenn man keinen Content einfügt.

s. https://www.beta.wordpress.rrze.fau.de/test/ Browserkonsole.

**Comment by xwolfde** [comment]
Ja, du hast recht. Aber dass kein Content eingegeben wurde, ist auch eher ein Fehler bei der EIngabe.

Gleichwohl iust das problem die Art der EIngabe: Wie soll die denn funktionieren, wenn man dem User keinerlei Hilfestellung gibt, wie er wo was eingeben soll. So hat niemand einen Plan, wie diese Scroll-Stories funktionieren sollen.

IMHO ist es daher vielleicht sinnvoller, die Scrollstorie-Template erstmal rauszunehmen und diese Funktionalität nur über Blöcke mit Gutenberg anzubilden. (Dann via dem Element-Blocks Plugin.).
Einverstanden?

**Comment by xwolfde** [comment]
@cassandre Nach Rücksprache mit Andi wollen wir diese Funktion vielleicht besser als Block im neuen Plugin nachbauen. Damut haben wir für den Autor eine klarere Usability bei der Eingabe, so dass man weiß was wie geschieht.


Bitte schau dir daher die Codes und JavaScripten von dieser Funktion an, inwieweit du sie in dem neuen Plugin übernehmen kann oder bau diese nach.

Relevant waren diese Skripten:
```
/ Enqueue necessary JavaScript files
function scroll_page_scripts() {
    wp_enqueue_script( 'd3', get_template_directory_uri() . '/js/fau-d3.v5.min.js', array( 'jquery' ), '1.0.0', true );
    wp_enqueue_script( 'scrollmagic',  get_template_directory_uri() . '/js/fau-ScrollMagic.min.js', array( 'jquery' ), '1.0.0', true );
    wp_enqueue_script( 'animation-gsap',  get_template_directory_uri() . '/js/fau-animation.gsap.min.js', array( 'jquery' ), '1.0.0', true );
    wp_enqueue_script( 'debug',  get_template_directory_uri() . '/js/fau-debug.addIndicators.min.js', array( 'jquery' ), '1.0.0', true );
    wp_enqueue_script( 'scroll-page-script', get_template_directory_uri() . '/js/fau-scroll-stories.min.js', array( 'jquery', 'd3', 'scrollmagic', 'animation-gsap', 'debug' ), '1.0.0', true );
}
add_action( 'wp_enqueue_scripts', 'scroll_page_scripts' );


```

Ich lass diese Skripten noch im Theme, were aber die Page Template Scroll-Stories erstmal wieder rausnehmen.
